x/bank/types/query.pb.go

```diff
@@ -10,7 +10,6 @@ import (
 	types "github.com/cosmos/cosmos-sdk/types"
 	query "github.com/cosmos/cosmos-sdk/types/query"
 	_ "github.com/gogo/protobuf/gogoproto"
-	grpc1 "github.com/gogo/protobuf/grpc"
```

Comment:
@aaronc what do I need to download to make this work?

Reply:
`make protoc-gen-gocosmos`. idk why it wasn't included on the `tool` cmd